modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Edges/DefaultEdge.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import React, {useMemo} from 'react';
+import {
+	EdgeProps,
+	EdgeText,
+	getEdgeCenter,
+	getSmoothStepPath,
+	useStoreState,
+} from 'react-flow-renderer';
+
+import {ObjectRelationshipEdgeData} from '../types';
+import {getEdgeParams} from '../utils';
+import ManyMarkerEnd from './ManyMarkerEnd';
+import OneMarkerEnd from './OneMarkerEnd';
+
+export default function DefaultEdge({
+	id,
+	source,
+	target,
+	style = {},
+	data,
+}: EdgeProps<ObjectRelationshipEdgeData>) {
+	const nodes = useStoreState((state) => state.nodes);
+
+	const sourceNode = useMemo(() => nodes.find((node) => node.id === source), [
+		source,
+		nodes,
+	]);
+	const targetNode = useMemo(() => nodes.find((node) => node.id === target), [
+		target,
+		nodes,
+	]);
+
+	if (!sourceNode || !targetNode) {
+		return null;
+	}
+
+	const {sourcePos, sx, sy, targetPos, tx, ty} = getEdgeParams(
```

Comment:
@igor-franca Please, rename those variables too. It will also be better to write it unabbreviated because we will be able to do the shorthand assignment in the rest of the file.